contracts/KeyManager.sol

```diff
@@ -171,28 +170,14 @@ contract KeyManager {
   }
 
   /**
-   * @dev Checks if sender is either the identity contract or a MANAGEMENT key
-   * @return `true` if sender is either identity contract or a MANAGEMENT key
+   * @dev Throws if called by any account other than a MANAGEMENT key.
    */
-  function _managementOrSelf()
-  internal
-  view
-  returns (bool found)
-  {
-    if (msg.sender == address(this)) {
```

Comment:
There is no scenario where the condition is true? Correct?

Reply:
I did not find any situation where we would need this at the moment